metadata.json

```diff
@@ -9,7 +9,7 @@
   "dependencies": [
     {
       "name": "puppetlabs/resource_api",
-      "version_requirement": ">= 1.0.0 < 2.0.0"
+      "version_requirement": ">= 1.6.3 < 2.0.0"
```

Comment:
Message seems to have deleted, but this cannot be updated as this refers to the module to help install the RSAPI on server/proxy and has only 1.0.0 as a released version

Reply:
Saw the message in email, reverted this